From 0934ff71af45e5454e381e0df36e5e0ded835f41 Mon Sep 17 00:00:00 2001
From: Alejandro Cerezo <alce65@hotmail.es>
Date: Wed, 15 Oct 2025 13:00:21 +0200
Subject: [PATCH] Solve bug in Home page js

---
 web/index.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/web/index.js b/web/index.js
index d2fbd64..0228efd 100644
--- a/web/index.js
+++ b/web/index.js
@@ -1 +1,2 @@
 console.log("Loading index.js");
+console.log("Solved initial bug")
-- 
2.51.0.windows.2

